pages/common/ghci.md

```diff
@@ -0,0 +1,11 @@
+# ghci
+
+> Interactive Haskell environment.
```

Comment:
We should add the words "Glasgow Haskell compiler" ..

Reply:
Perhaps `Interactive Glasgow Haskell environment.` would work?

Reply:
Ah! I see. `Interactive Glasgow Haskell Compiler environment.` then?